packages/survey/stories/ItemGroup.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import { Meta, Story } from '@storybook/react/types-6-0'
+
+import { ItemGroup, ItemGroupProps } from '../src/Question/ItemGroup'
+
+export default {
+  title: 'Components/Survey/Question/ItemGroup',
+  component: ItemGroup,
+  argTypes: {
+    activeColor: {
+      control: 'color',
+    },
+  },
+  args: {
+    name: 'my_items_group',
+    itemType: 'radio',
+    options: [
+      { value: 'value_1', label: 'Value 1' },
+      { value: 'value_2', label: 'Value 2' },
+      { value: 'value_3', label: 'Value 3' },
+    ],
+    direction: 'vertical',
+    withLabels: true,
+    activeColor: '#990033',
+  },
```

Comment:
This `args` object should only contain parameters that are **required**.
This way we can see what is rendered in the default case when no optional parameters are set.

Reply:
done!